Vagrantfile

```diff
@@ -26,7 +26,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # using a specific IP.
   config.vm.network "private_network", ip: "192.168.33.200"
   config.vm.network :forwarded_port, guest: 8100, host: 8100
-
+  config.vm.provider "virtualbox" do |vb|
+    vb.customize ["modifyvm", :id, "--memory", "1024"]
+  end
```

Comment:
Yeah, It probably isn't strictly required, but I OOMed the box doing the minify with uglify-js